### ISSUE TITLE
Is the version necessary for Chocolately?

### ISSUE THREAD

**Issue by mraible** [okta/okta-cli#63]
I noticed that most of the install commands in the README don't require a version. However, the Chocolately example does use a version:

```
choco install okta --version=0.7.0
```

Is this necessary? If so, we might want to automate updating the README as part of the release process.

**Comment by bdemers** [comment]
Yes and no, our Chocolately packages are currently manually moderated, sometimes it takes a couple of weeks for that to happen.  Using the `version` flag is the workaround. However, we can (and probably should) remove the flag after the manual moderation step.

**NOTE:** There are a bunch of automated steps too, that block invalid/bad packages too.

**Comment by mraible** [comment]
@bdemers Did you get any more info on this in your Twitch stream today?

**Comment by bdemers** [comment]
Basically, it's a human moderation delay, once we have a few more versions out, it's _possible_ that it could be automated (which probably means hours or days instead of weeks).  So for now, we are the odd situation of, after a release the `--version` flag is needed for a few weeks.
